common/src/main/java/muramasa/gregtech/data/TierMaps.java

```diff
@@ -107,21 +107,21 @@ public static void providerInit() {
         {
             ImmutableMap.Builder<Tier, PipeItemBlock> builder = ImmutableMap.builder();
             builder.put(Tier.ULV, WIRE_LEAD.getBlockItem(PipeSize.VTINY));
-            builder.put(Tier.LV, WIRE_TIN.getBlockItem(PipeSize.VTINY));
-            builder.put(Tier.MV, WIRE_COPPER.getBlockItem(PipeSize.VTINY));
-            builder.put(Tier.HV, WIRE_GOLD.getBlockItem(PipeSize.TINY));
-            builder.put(Tier.EV, WIRE_ANNEALED_COPPER.getBlockItem(PipeSize.SMALL));
-            builder.put(Tier.IV, WIRE_PLATINUM.getBlockItem(PipeSize.TINY));
+            builder.put(Tier.LV, WIRE_COPPER.getBlockItem(PipeSize.VTINY));
```

Comment:
Why was this changed to copper? Copper is tier 2, not tier 1

Reply:
because I wanted to not use the same material for cables and wires. in that process, I accidentally used an mv wire for lv. will change it to another 32 EU wire asap